XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -52,7 +56,7 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
-  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp')) ">
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp')) AND '$(OS)' == 'Windows_NT' ">
```

Comment:
netcoreapp targets if '$(OS)' == 'Windows_NT' (see TargetFrameworks), I think this changes are redundant

Reply:
looks weird. on Mac this section should be ignored at all. How does it work on CI?

Reply:
It passes all the tests

Reply:
it passes all the tests even without your changes. I am OK with this changes but it would be nice to understand why does it fails on your env and passes on CI

Reply:
@VladislavAntonyuk I think the CI runs on Windows :) https://github.com/xamarin/XamarinCommunityToolkit/blob/main/azure-pipelines.yml#L49

Reply:
But .net core should run on macOS https://github.com/xamarin/XamarinCommunityToolkit/blob/88ef7f9b0fc9d7db170f40870b5627ff7f680a65/azure-pipelines.yml#L110

Reply:
I wouldn't be surprised if that was a bug of sorts